docs/quick_start.md

```diff
@@ -25,7 +25,7 @@ Note: Beginning in Helm v3.8.0, OCI support is enabled by default, and it gradua
 
 ```bash
 helm upgrade --install lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version]
-  
+
```

Comment:
Please remove useless blank lines to avoid conflicts with other pr merges

Reply:
> Please remove useless blank lines to avoid conflicts with other pr merges

done